.github/ISSUE_TEMPLATE/rebase.md

```diff
@@ -1,5 +1,13 @@
 # Rebase to a new version of Fedora (N)
 
+## At previous Fedora major release
+
+### Open tickets to track related work for this release
+
+- [ ] Fedora Changes Considerations [example](https://github.com/coreos/fedora-coreos-tracker/issues/918)
+- [ ] Package Additions/Removals [example](https://github.com/coreos/fedora-coreos-tracker/issues/1128)
+- [ ] Test Week [example](https://github.com/coreos/fedora-coreos-tracker/issues/1123)
```

Comment:
Could we do this at the end of the checklist instead for the next rebase? (I.e. put it under the "Open ticket for the next Fedora rebase" section.)

Reply:
I started with them down there but then moved them up here. I decided to do that because it's nicer to be able to fire off and create a single ticket (that has a full template) than it is to do these other tickets (require a little more thought). i.e. someone can finish the previous release easily. These other tickets can get picked up later (they are tracked at least).